add-raster-from-service/README.md

```diff
@@ -0,0 +1,44 @@
+# Add raster from service
+
+Load a raster from a service, then apply a function to it.
+
+![Image of add raster from service](add-raster-from-service.png)
+
+## Use case
+
+Raster functions allow processing operations that can be applied to one or more rasters on the fly. Functions can be applied to rasters that come from a service. A land survey agency may apply hillshade and aspect functions to rasters with elevation data in order to better determine the topography of a landscape and to make further planning decisions.
+
+## How to use the sample
+
+Tap the "Apply Layer" button to apply the raster function and display the result.
```

Comment:
we could also, add about the `reset` button which applies the image service raster back to the rasterLayer